### PR TITLE
(BIDS-1972) lets trim the balance

### PR DESCRIPTION
**Pull request by MauserBitfly**
### <samp>🤖 Generated by Copilot at 2661991</samp>

Refactored and enhanced the currency formatting logic in `utils/format.go`. Increased the accuracy of exchanged amounts in the transaction page.

### CODE REVIEW

File: utils/format.go [hunk 1]
```diff
@@ -361,16 +361,7 @@ func FormatTransactionType(txnType uint8) string {
...
 func FormatCurrentBalance(balanceInt uint64, currency string) template.HTML {
-	if currency == "ETH" {
-		exchangeRate := ExchangeRateForCurrency(currency)
-		balance := float64(balanceInt) / float64(1e9)
-		return template.HTML(fmt.Sprintf("%.5f %v", balance*exchangeRate, currency))
-	} else {
-		exchangeRate := ExchangeRateForCurrency(currency)
-		balance := FormatFloat((float64(balanceInt)/float64(1e9))*float64(exchangeRate), 2)
-
-		return template.HTML(fmt.Sprintf(`%s %v`, balance, currency))
```
Comment (Eisei24) [inline utils/format.go:372]: The currency is now missing for the balance, you can compare your implementation with **FormatExchangedAmount**.

**Review by Eisei24 — CHANGES_REQUESTED** [review]
Please fix the missing currency display.

File: utils/format.go (hunk 1 above)
Comment (MauserBitfly) [reply utils/format.go:372]: added it back

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**